ui-tests/src/test/java/org/openmrs/reference/page/PatientVisitsDashboardPage.java

```diff
@@ -31,6 +31,11 @@
     private static final By VISIT_NOTE_ENCOUNTER = By.xpath("//div[@id='visit-details']/ul/li/ul/li/div/strong/span[text()='Visit Note']");
     private static final By VISIT_NOTE = By.id("referenceapplication.realTime.simpleVisitNote");
     private static final By RETURN_TO_DASHBOARD = By.xpath("//*[@id='breadcrumbs']/li[2]/a");
+    private static final By EDIT_VITALS_ICON = By.cssSelector("#encountersList > li > span > i.editEncounter.edit-action.icon-pencil");
+    private static final By VIEW_VITALS_ICON  = By.cssSelector("#encountersList > li > span > i.viewEncounter.view-action.icon-file-alt");
+    private static final By DELETE_VITALS_ICON = By.cssSelector("#encountersList > li > span > i.deleteEncounterId.delete-action.icon-remove");
+    private static final By COMFIRM_DELETE_BTN  = By.cssSelector("#delete-encounter-dialog > div.dialog-content > button.confirm.right");
+
```

Comment:
Why not using the word CONFIRM_DELETE_BUTTON?

Any reason of adding another empty code line at line `38` ?

I think it's cleaner to user 
@kdaud I think it's cleaner to do this:

```js
{
EDIT_VITALS_ICON = By.cssSelector("#encountersList i.editEncounter.edit-action.icon-pencil");

VIEW_VITALS_ICON  = By.cssSelector("#encountersList i.viewEncounter.view-action.icon-file-alt");

DELETE_VITALS_ICON = By.cssSelector("#encountersList i.deleteEncounterId.delete-action.icon-remove");

COMFIRM_DELETE_BUTTON  = By.cssSelector("#delete-encounter-dialog button.confirm.right");
  
  }
```

Reply:
I just wanted to use btn because it is a short form of button.  Does it cause any harm @kdaud ? 
line 38 just skipped my sight, I will remove it.

Reply:
> I just wanted to use btn because it is a short form of button

Why using short form of the word ? Have you ever had a look at this interpretation [here](https://www.google.com/search?q=btn+meaning+in+text&oq=BTN+meaning&aqs=chrome.1.0i512l6j0i22i30l4.11905j0j7&sourceid=chrome&ie=UTF-8) for BTN?

Reply:
:smile: @kdaud . I will work on that!

Reply:
@Irenyak1 did you take a look at the third comment [above](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/562#discussion_r709296827)?

Reply:
> @Irenyak1 did you take a look at the third comment [above](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/562#discussion_r709296827)?

I have not understood what you need me to do in this comment. Could you please explain

Reply:
@Irenyak1 I think it's a good idea to use cleaner element reference for the page elements. Something like;

```js
{
 private static final By EDIT_VITALS_ICON = By.cssSelector("#encountersList i.editEncounter.edit-action.icon-pencil");

 private static final By VIEW_VITALS_ICON  = By.cssSelector("#encountersList i.viewEncounter.view-action.icon-file-alt");

 private static final By DELETE_VITALS_ICON = By.cssSelector("#encountersList i.deleteEncounterId.delete-action.icon-remove");

 private static final By COMFIRM_DELETE_BUTTON  = By.cssSelector("#delete-encounter-dialog button.confirm.right");
  
  }
```

Reply:
Ah ok, I get you now @kdaud